docs/pages/enroll-resources/application-access/cloud-apis/aws-console-without-agent.mdx

```diff
@@ -0,0 +1,379 @@
+However, it does not record the requests in Teleport's audit logs, only that a session was started by a given user.
+</Admonition>
+
+This guide will explain how to:
+
+- Configure AWS Console and CLI access with Teleport using Roles Anywhere
+- Access the AWS Management Console
+- Access the AWS Command Line Interface (CLI)
+- Access applications using AWS SDKs
+
+## How it works
+
+Teleport leverages AWS IAM Roles Anywhere to issue temporary credentials for assuming target IAM Roles.
+
+Access is managed through Teleport's RBAC policies, ensuring credentials are only generated for authorized users and roles.
+
+For web console access, you can navigate to the resources page in Teleport Web UI and click on the AWS Application which is named after the Roles Anywhere Profile.
+
+For CLI and SDK based access, you must use `tsh` to obtain the credentials
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- Permissions in AWS to:
+  - create IAM Roles Anywhere Trust Anchor
+  - create IAM Roles Anywhere Profiles
+  - create IAM Roles
+- `aws` command line interface (CLI) tool in PATH. Read the AWS documentation to
+  [install or update the latest version of the AWS
+  CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+## Step 1/4. Configure AWS IAM Roles Anywhere integration
+
+In this section, you will configure AWS IAM resources to allow Teleport to issue AWS credentials.
+
+<Admonition type="note">
+Instead of doing this step manually, you can navigate to your Teleport Web UI,
+click on Enroll New Resource in Resources listing page,
+and follow the guide after clicking on the AWS CLI/Console Access tile.
+</Admonition>
+
+You will create the following resources in AWS:
+
+| Name                                 | Resource                        | Function                                                                                  |
+|--------------------------------------|---------------------------------|-------------------------------------------------------------------------------------------|
+| `TeleportRolesAnywhereIntegrationCA` | IAM Roles Anywhere Trust Anchor | Allow access from Teleport into AWS.                                                      |
+| `TeleportRolesAnywhereProfileSync`   | IAM Role                        | Allows Teleport to iterate over AWS Roles Anywhere Profiles and import them as resources. |
+| `TeleportRolesAnywhereProfileSync`   | IAM Roles Anywhere Profile      | Allows access to `TeleportRolesAnywhereProfileSync` IAM Role.                             |
+
+### Create an IAM Roles Anywhere Trust Anchor
+
+First, you will create an IAM Roles Anywhere Trust Anchor which trusts the Teleport's AWS Roles Anywhere CA.
+
+1. Obtain the Teleport certificate by:
+  <Tabs>
+    <TabItem label="Via curl">
+    ```code
+    $ curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=awsra'
```

Comment:
Does this export multiple certificates in clusters with multiple active signers (like those using HSMs)?

Reply:
No, in that case the curl receives an error from the server.
https://github.com/gravitational/teleport/blob/e3615114b631ea8dc64129f9a2f68a29fe571531/lib/web/ca_export.go#L85-L87

To fix this we can:
- expect the user to use tctl
- expect the user to read the user message and have them add the `format=json|zip` and then user other tools to extract the CAs and paste them in AWS
- change the simple `curl` command to something more elaborate that will work whether they have just one or multiple active signers

We can do the 3rd option, but that requires a dependency on `jq`.
What do you think?

```bash
$ curl 'https://<Var name="teleport.example.com" />/webapi/auth/export?type=awsra&format=json' | jq -r '[].data | @base64d'